cmd/operator-sdk/build/cmd.go

```diff
@@ -65,7 +60,7 @@ For example:
 		"Tool to build OCI images. One of: [docker, podman, buildah]")
 
 	// todo: remove when the legacy layout is no longer supported
-	if !kbutil.HasProjectFile() {
+	if !projutil.HasProjectFile() {
```

Comment:
It should be removed. All projects for 1.0.0 should have HasProjectFile
The code inside of this checks is related to legacy layout.